### PR TITLE
[BUGFIX][Driver] HDFS config from ZK fix

### PR DESCRIPTION
**Pull request by sgomezg**
HDFS config from ZK fix

### CODE REVIEW

**Comment by anistal** [comment]
+1

File: serving-api/src/main/resources/reference.conf [hunk 1]
```diff
@@ -19,14 +19,14 @@ sparkta {
   }
 
   config {
-    executionMode = local
+    executionMode = yarn
```
Comment (danielcsant) [inline serving-api/src/main/resources/reference.conf]: could you set this property as local?

**Comment by danielcsant** [comment]
Please update zookeeper property in doc:

https://stratio.atlassian.net/wiki/display/SPARKTA0x9/6.+Configuration

**Comment by sgomezg** [comment]
Done, thanks :)

**Comment by danielcsant** [comment]
+1

**Pull request merged**